python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -74,6 +73,7 @@ class Node(object):
     thread-safe usage.
     """
     __metaclass__ = abc.ABCMeta
+    __slots__ = ('_index_cache_dict', '_parent', '_read_lock', '_write_lock', '_dirty', '_registry', '_data_dict')
```

Comment:
OK, just turning this __off__ I see 324Mb consumed when the list from `%time dataset = BKQuery('/LHCb/Collision15/Beam6500GeV-VeloClosed-MagDown/Real Data/Reco15a/Stripping24/90000000/EW.DST').getDataset()` is loaded
but when I turn this __on__ I get 262Mb. Both times clean session with no objects loaded other than this and monitoring turned off.
This is a big enough change that I'm for keeping this in